stories/Molecules/FooterNavigation/FooterIcons/FooterIcons.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './footerIcons.scss';
+import { Image } from '../../../Atom/Images/Image/Image';
+
+
+export const FooterIcons = ({ style }) => {
+  return (
+    <ul className={["footer__icons", `footer__icons--${style}`].join(" ")}>
+      <a href="#" className="facebook"></a>
+      <a href="#" className="linkedIn"></a>
+      <a href="#" className="instragram"></a>
+      <a href="#" className="twitter"></a>
+      <a href="#" className="youtube"></a>
```

Comment:
@harishkumar-srijan it is semantically wrong...please use li tag inside the ul tag